parquet/src/file/metadata.rs

```diff
@@ -940,6 +974,48 @@ impl OffsetIndexBuilder {
     }
 }
 
+/// Metadata for sorting order for columns
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct SortingColumnMetaData {
```

Comment:
```suggestion
pub struct SortingColumn {
```

I think it is better to use the same name as the parquet docs

Reply:
I followed the current notation of having two structs `RowGroup` for thrift read/write and `RowGroupMetaData` in `FieldMetaData`. `SortingColumn` is used in thrift read/write and `SortingColumnMetaData` in `FileMetaData`. I am going to keep this as is.

Reply:
That is because they are actually metadata, i.e. RowGroupMetaData is actually the metadata for the RowGroup, the thrift name is a little strange imo. Calling this `SortColumn` is consistent with `Statistics`, `Encoding`, `ConvertedType`,`LogicalType`, `PageType` etc...